### ISSUE TITLE
Switchbot hub Remote control not working properly

### ISSUE THREAD

**Issue by ahuygens1994** [AdyRock/com.switchbot#147]
I have used a Switchbot mini and Hub 2 as a remote control for my LG airconditioning. Both hubs are for similar model of airconditioning.
Before I was able to switch off the airconditioning by pushing the power off button in my Homey Pro app, but it gives the “Power on” command. In the Switchbot app the controls all work fine.
Upon checking, all the commands given in the Homey Pro app only forward the “Power On” command, which means I am unable to control my airconditioning from my Homey Pro app.

**Comment by emi662002** [comment]
Same issue, I'd like to add: I have homey linked to Google home. The airco device synced from homey to Google assistant work fine, but the openapi from homey shows this issue. 
I've updated the firmware of the mini hub and it didn't help.

**Comment by faahrev** [comment]
Same here: all the commands given in the Homey Pro app only forward the “Power On” command, which means I am unable to control my airconditioning from my Homey Pro app. Using 4 Mini Hubs that only switch my airconditioning units on since a couple of days. No problem using the Switchbot app.

**Comment by remy561** [comment]
@faahrev, you’re right. Turn on also doesn’t work anymore. Its always sending an ‘empty on’ command which results in a cool 25c on my device, resulting in it just blowing wind.

@everyone, you can work around this issue for now by using Switchbot scenes. For example, my day 19c / night 16c heating flow became:
![IMG_0311](https://github.com/user-attachments/assets/49ea644d-f21d-4088-8f71-75ae6b03cf7e)

Lets hope it gets resolved soon 🙂

**Comment by AdyRock** [comment]
Sorry guys, my wife was diagnosed with bowl cancer and has just had an operation to remove it, so my mind is on other things at the moment. 
Has anyone sent a log from the app settings page, preferably with the Log level set to Full information when sending a remote command?

I have asked my contact at SwitchBot to verify their end.

**Comment by remy561** [comment]
@AdyRock, oof that must be one heck of a tense situation. You should indeed not be focussing on the app, especially now we can use the Scene workaround. Best wishes to you and your family ❤️‍🩹

That said, I’ve pressed some buttons and sent you the log. The Homey log all looks fine, but the Switchbot one is full of errors. 
![IMG_0312](https://github.com/user-attachments/assets/8a11eb89-9435-4c72-a75f-bc5f35d70010)

Take your time,
Remy

**Comment by AdyRock** [comment]
Looking at the log, Homey seems to be sending the correct commands, so I think the SwitchBot server is not decoding it properly. I have referenced this issue with my contact, so hopefully, he can check their systems.

**Comment by ahuygens1994** [comment]
I understand that this is not your priority at the moment! Completely understandable!
I wish you and your family a lot of strength during these challenging times.

But I can also brung you some good news. I just checked if it would work again through Homey, and it did!
I am able to switch my AC’s on and off again, change temperature. So it looks like it’s working fine again I guess.

**Comment by faahrev** [comment]
Indeed. Mine are responding again too.
Anyone with information about the cause?

**Comment by chenliuyun** [comment]
Hi, guys, this is from a SwitchBot developer, who has confirmed that this is a server bug that was fixed on the 20th. You are welcome to submit relevant logs through the app feedback if you have any issues, and we will review them promptly.
